### PR TITLE
Connect to spark-ui running on Driver Pod

### PR DESCRIPTION
**Pull request by awsalialem**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/debugging/docs/connect-spark-ui.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# **Connect to Spark UI running on the Driver Pod**
+
+To obtain more detail about their application or monitor their job execution, Spark application developers can connect to Spark-UI running on the Driver Pod.
```
Comment (nithu0115) [inline content/debugging/docs/connect-spark-ui.md:3]: Could you add a statement that, Spark UI (Spark history server) is packaged with EMR on EKS out of the box. Alternatively, if customers want to see Spark UI immediately after the driver is spun up, they can use `kubectl port-forward` route?

File: content/debugging/docs/connect-spark-ui.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        "logUri": "s3://joblogs"
+      }
+    }
+  }
+}
+EOF
+aws emr-containers start-job-run --cli-input-json file:///spark-python.json
+```
+
+Once the job is submitted successfully, run `kubectl get pods -n <virtual-cluster-k8s-namespace> -w ` command to watch all the pods, until you observe the driver pod is in the "Running" state. The Driver pod's name usually is in `spark-<job-id>-driver` format.
```
Comment (nithu0115) [inline content/debugging/docs/connect-spark-ui.md:56]: Could you add a pre-requisite section:
* AWS cli should be installed
* kubectl should be installed
* Customer should run `aws eks update-kubeconfig --name <cluster--name> --region <us-west-2> to download kubeconfig file and use currect context to talk to API server.

**Review by nithu0115 — CHANGES_REQUESTED** [review]
Thanks for your contribution @awsalialem! Overall, PR looks good! Some minor comments...

**Comment by awsalialem** [comment]
Addressed the change requests, appreciate another review

File: content/debugging/docs/connect-spark-ui.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Spark UI (Spark history server) is packaged with EMR on EKS out of the box. Alternatively, if you want to see Spark UI immediately after the driver is spun up, you can use the instructions in this page to connect.
+
+This page shows how to use `kubectl port-forward` to connect to the Job's Driver Pod running in a Kubernetes cluster. This type of connection is useful for debugging purposes.
+
+**Pre-Requisites**
+
+* AWS cli should be installed
+* "kubectl" should be installed
+* If this is the first time you are connecting to your EKS cluster from your machine, you should run `aws eks update-kubeconfig --name --region` to download kubeconfig file and use currect context to talk to API server.
```
Comment (dacort) [inline content/debugging/docs/connect-spark-ui.md]: Spelling.

```suggestion
* If this is the first time you are connecting to your EKS cluster from your machine, you should run `aws eks update-kubeconfig --name --region` to download kubeconfig file and use correct context to talk to API server.
```

File: content/debugging/docs/connect-spark-ui.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+
+### Connecting to the Driver Pod
+
+Spark Driver Pod hosts Spark-UI on port `4040`. However the pod runs within the internal Kubernetes network. To get access to the internal Kubernetes resources, `kubectl` provides a tool ("Port Forwarding") that allows access from your localhost. To get access to the driver pod in your cluster:
+
+
+1- Run ```kubectl port-forward <driver-pod-name> 4040:4040```
+
+The result should as the following:
```
Comment (dacort) [inline content/debugging/docs/connect-spark-ui.md]: Grammar.
```suggestion
The result should be the following:
```

**Review by dacort — CHANGES_REQUESTED** [review]
2 more minor revisions.

**Comment by awsalialem** [comment]
Addressed the spelling errors

**Review by dacort — APPROVED** [review]

**Pull request merged**